src/tests/ftest/util/ec_utils.py

```diff
@@ -61,6 +61,8 @@ def check_aggregation_status(pool, quick_check=True, attempt=20):
                 # Return immediately once aggregation starts for quick check
                 if quick_check:
                     return agg_status
+            else:
+                initial_usage[storage_type] = current_usage[storage_type]
```

Comment:
Optional - Recommend renaming this variable to `previous_usage` since it's no longer only the initial

Reply:
ok, may rename it in following patch to avoid test it again. thx